_rules/letter-spacing-not-important-24afc2.md

```diff
@@ -0,0 +1,271 @@
+
+```html
+<p style="letter-spacing: 0.1em">
+	<span style="letter-spacing: inherit !important;">
+		The toy brought back fond memories of being lost in the rain forest.
+	</span>
+</p>
+```
+
+#### Passed Example 8
+
+Both this `p` and `span` elements match the **not important** condition. For the `span`, the [cascaded][] value is `unset !important`, which is equivalent as `inherit` since it is an inherited property, thus the [computed][] value is the [inherited][] value, that is the [computed][] value of its parent, and it is not [important][]. Even though the value is too small, styles with [author origin][] declared by assistive technologies may win the [cascade sort][] and override it, thus this may satisfy [Success Criterion 1.4.12 Text Spacing][sc1412] and does not fail this rule. This is nonetheless bad practice and sufficient spacing should be used.
+
+```html
+<p style="letter-spacing: 0.1em">
+	<span style="letter-spacing: unset !important;">
+		The toy brought back fond memories of being lost in the rain forest.
+	</span>
+</p>
+```
+
+### Failed
+
+#### Failed Example 1
+
+This `p` element has a [computed][] `letter-spacing` of `1.6px` (assuming a default `medium` font size of `16px`) which is below the recommended minimum.
+
+```html
+<p style="letter-spacing: 0.1em !important">
+	The toy brought back fond memories of being lost in the rain forest.
+</p>
+```
+
+#### Failed Example 2
+
+This `p` element has a [computed][] `letter-spacing` of `2px` which is only 0.1 times the font size (`20px`), thus below the recommended minimum.
+
+```html
+<style>
+	p {
+		font-size: 20px;
+	}
+</style>
+
+<p style="letter-spacing: 2px !important">
+	The toy brought back fond memories of being lost in the rain forest.
+</p>
+```
+
+#### Failed Example 3
+
+This `p` element has a [computed][] `letter-spacing` of 0.
+
+```html
+<p style="letter-spacing: normal !important">
+	The toy brought back fond memories of being lost in the rain forest.
+</p>
+```
+
+#### Failed Example 4
+
+This `p` element has a [computed][] `letter-spacing` of 0.
+
+```html
+<p style="letter-spacing: initial !important">
+	The toy brought back fond memories of being lost in the rain forest.
+</p>
+```
+
+### Inapplicable
+
+#### Inapplicable Example 1
+
+There is no HTML element.
+
+```svg
+<svg xmlns="http://www.w3.org/2000/svg"></svg>
+```
```

Comment:
I don't think SVG is necessary in this.

Reply:
I think most (all?) our rules targeting HTML elements have a non-HTML element has inapplicable example.

There's a bit of the same with the two invisible examples where maybe one would be enough…

Reply:
Not that I'm aware of.... Rules that apply to HTML elements have SVG inapplicable examples. But I think this is a little different.

I do think the two invisible examples are needed. Off-screen is fairly different from display:none.

Reply:
Fair, I suppose... But then shouldn't this have a text node with a style attribute with a letter-spacing property? Otherwise what are you testing?